package.json

```diff
@@ -4,7 +4,9 @@
   "main": "server/index.js",
   "scripts": {
     "server-dev": "npx nodemon --watch server server/index.js",
-    "client-dev": "npx webpack serve --hot --open",
+    "client-dev": "cross-env NODE_ENV=development npx webpack serve --hot --open --config webpack.dev.js",
```

Comment:
I take it that the changes here were to get webpack to build the bundle differently if its in dev or production?